### DISCUSSION TITLE
Error response from daemon: Get "": tls: failed to verify certificate: x509: certificate signed by unknown authority

### DISCUSSION THREAD

**Discussion by haleviyi** [actions/actions-runner-controller#3198] [Questions]
Hi,

I am trying to run ARC with dind mode, but it failed on docker pull from our private registry when starting job container:
 Error response from daemon: Get "": tls: failed to verify certificate: x509: certificate signed by unknown authority.
  
Any idea how to solve it?

**Comment by haleviyi** [comment]
Is it possible to use private registry while using dind mode?

**Comment by haleviyi** [comment]
I created a docker image with the certificates from the docker:dind image and it solve the issue.

**Reply by AurimasNav** [reply]
I was mounting certs into the wrong container, once I mounted them into dind container everything went smoothly, thanks for the pointer.
